lib/collection/src/problems/too_many_collections.rs

```diff
@@ -0,0 +1,61 @@
+use issues::{Code, Issue, Solution};
+
+pub struct TooManyCollections;
+
+impl Issue for TooManyCollections {
+    fn instance_id(&self) -> &str {
+        "" // Only one issue for the whole app
```

Comment:
Would it make sense to change this to something like `too_many_collections`, or is it purely to distinguish between issues of this specific type?